src/packages/searchbar/searchbar.taro.tsx

```
@@ -87,21 +91,20 @@ export const SearchBar: FunctionComponent<
     const searchSelf: HTMLInputElement | null = searchRef.current
     searchSelf && searchSelf.focus()
   }
-  const change = (event: ChangeEvent<HTMLInputElement>) => {
-    if (value === event.target.value) return
-    onChange && onChange?.(event.target.value, event)
-    setValue(event.target.value)
-    event.target.value === '' && forceFocus()
+  const onInput = (event: any) => {
+    const eventValue = event?.detail?.value
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)